api/src/main/java/io/strimzi/api/kafka/model/AclRuleTransactionalIDResource.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+/**
+ * A representation of a transactional ID resource for ACLs
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"type", "name", "patternType"})
+public class AclRuleTransactionalIDResource extends AclRuleResource {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_TRANSACTIONAL_ID = "transactionalID";
+
+    private String name;
```

Comment:
Do we really need the name and patternType fields here? They do not seem to be used.

Reply:
Yes, we do need them.